BlogTemplate.Tests/Pages/EditTests.cs

```diff
@@ -43,5 +43,101 @@ public void UpdatePostP2P_TitleIsUpdated_UpdateSlug()
 
             Assert.Equal("Edited-Title", post.Slug);
         }
+        [Fact]
+        public void UpdatePostD2P_TitleIsUpdated_UpdateSlug()

```

Comment:
As I mentioned in previous review (which I submitted as you pushed this), expand D2P and separate it from the name of the method being tested.